### DISCUSSION TITLE
Map hintergründe

### DISCUSSION THREAD

**Discussion by ChillLP** [marhei/TrainCompany-Data#132] [Ideas]
Auswahl aus verschiedenen hintergründen, damit orientierung af der karte vllt einfacher fällt, z.b. Openrailwaymaps oder auch nur Google Maps / Earth.

**Comment by marhei** [comment]
Gute Idee, aber ich hab überhaupt keine Ahnung wie man sowas am besten machen könnte 😵‍💫

**Reply by C1710** [reply]
Mh, es wird doch eine SVG-Datei generiert, richtig? Dann ließe sich doch theoretisch einfach eine Karte als Ebene hinzufügen?
Wäre nicht zwingend schön, weil die nur eine Detailstufe hätte, aber es wäre glaub ich relativ einfach 🤷‍♀️

**Comment by ChillLP** [comment]
Dann sind wir schonmal 2

**Comment by C1710** [comment]
Würde sowas wohl helfen?
![Karte mit Bundesländern als Hintergrund](https://user-images.githubusercontent.com/10349490/177036399-e885ccd2-0578-4e58-88b4-35e32b587675.png)

**Reply by marhei** [reply]
Naja, wäre immerhin ein Anfang. :D Würde vielleicht die farbige Hinterlegung der Bundesländer eher unterlassen. Am wichtigsten: Grenzen und Gewässer.

**Reply by C1710** [reply]
![Gleiche Karte, ohne Bahnnetz und mit weniger Farbe](https://user-images.githubusercontent.com/10349490/177135859-19102781-c219-4293-9fe2-233d97e02b67.png)

So sind die Farben noch ganz minimal da...

**Reply by C1710** [reply]
Die Karte hat nur das Problem, dass sie mit den internationalen Erweiterungen problematisch werden könnte

**Comment by f2k1de** [comment]
https://commons.wikimedia.org/wiki/File:Central_Europe_location_map.svg
https://commons.wikimedia.org/wiki/File:D-A-CH_location_map.svg

**Comment by C1710** [comment]
Ich habe mal das weiße Overlay durch Transparenz ersetzt. Das sollte dann im Darkmode besser aussehen:
SVG:
![Germany,_administrative_divisions_(+districts)_-_de_-_colored](https://user-images.githubusercontent.com/10349490/177518504-8febbdda-6322-4d53-b858-14695394dfec.svg)
PNG:
![Germany,_administrative_divisions_(+districts)_-_de_-_colored](https://user-images.githubusercontent.com/10349490/177518521-18a0cd52-080a-45b6-81dd-25a37c244804.png)

**Comment by C1710** [comment]
Hab jetzt mal die Central Europe-Karten angepasst.
Offsets: x: 593,  y: 630
PNG:
![Central_Europe_location_map](https://user-images.githubusercontent.com/10349490/178110345-472919ff-6ce2-4008-b003-9a9b516f2b27.png)

SVG:
![Central_Europe_location_map](https://user-images.githubusercontent.com/10349490/178110350-5c48b363-baef-4028-aa51-04474e4409fd.svg)

**Comment by C1710** [comment]
Hier noch in nicht gelb:
PNG:
![Central_Europe_location_map](https://user-images.githubusercontent.com/10349490/178110579-938d4a0f-a0d2-4377-8edf-0b1001835455.svg)

SVG:
![Central_Europe_location_map](https://user-images.githubusercontent.com/10349490/178110578-5e3aacb8-2d86-476e-a83c-396f007643b3.png)

**Comment by C1710** [comment]
Mit der Europa-Karte (ohne Änderung der Verhältnisse usw.:
PNG:
![Europe_blank_laea_location_map](https://user-images.githubusercontent.com/10349490/178164009-52e24d40-f1e8-4563-b850-e83a09322821.png)
SVG:
![Europe_blank_laea_location_map](https://user-images.githubusercontent.com/10349490/178164014-ac5323a5-44d7-4062-816a-15c69917b56b.svg)

**Comment by marhei** [comment]
Wo Angepasste Karte für GB? 👀

**Comment by C1710** [comment]
Die Karte funktioniert erst mit den neuen Koordinaten.
Offsets: x: 2730, y: 3418

Weil die als PNG in einer akzeptablen Version sehr groß würde, nur als gezippte SVG:
[TC-Karte-Neu.zip](https://github.com/marhei/TrainCompany-Data/files/9110346/TC-Karte-Neu.zip)

**Reply by marhei** [reply]
Ich danke dir! :)
